src/server.c

```diff
@@ -1140,6 +1187,16 @@ void nvnc_close(struct nvnc* self)
 	LIST_FOREACH_SAFE (client, &self->clients, link, tmp)
 		client_unref(client);
 
+	struct sockaddr_storage addr;
+	struct sockaddr_un *addr_unix;
+	socklen_t addr_len = sizeof(addr);
+	if (!getsockname(self->fd, (struct sockaddr*)&addr, &addr_len)) {
```

Comment:
This project compares against zero to check for errors. I.e. we use `== 0`, not `!`.